### ISSUE TITLE
App crash on devices with Android version less than 23

### ISSUE THREAD

**Issue by ozaneb** [skydoves/Balloon#354]
**Please complete the following information:**
- Library Version 1.4.6
- Affected Device(s) emulator device with Android 5.1
 
**Describe the Bug:**

Following crash occurs while trying to show balloon on devices with Android sdk version 22 and older. 

```
    java.lang.NoSuchMethodError: No virtual method setForeground(Landroid/graphics/drawable/Drawable;)V in class Landroidx/appcompat/widget/AppCompatImageView; or its super classes (declaration of 'androidx.appcompat.widget.AppCompatImageView' appears in /data/app/nozy.video.livestream-1/base.apk)
        at com.skydoves.balloon.Balloon.initializeArrow$lambda-5$lambda-4(Balloon.kt:286)
        at com.skydoves.balloon.Balloon.$r8$lambda$6QerMEq1X1Gznc0_SUWDE5bxR6I(Balloon.kt)
        at com.skydoves.balloon.Balloon$$ExternalSyntheticLambda6.run(D8$$SyntheticClass)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5254)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```

Apparently, `View.setForeground` method added in api level 23 (https://developer.android.com/reference/android/view/View#setForeground(android.graphics.drawable.Drawable)), but the documentation claims that the library supports API 17+.  I see "NewApi" lint warnings were suppressed :)

**Expected Behavior:**

Balloon library should not cause a crash because of usage of `View.setForeground` method on devices with API Level greater than 16.

**Comment by ozaneb** [comment]
Hello @skydoves, I am using latest stable version `1.4.2`. Upon your suggestion, I tried again using app compat `1.5.0-rc1`, nevertheless the same crash happened.

**Comment by skydoves** [comment]
Could you try again with `1.4.6`? The recent version doesn't use the `AppCompatImage` internally.

**Comment by skydoves** [comment]
Hey @ozaneb, could you try to build with the new snapshot version below?

To import snapshot versions on your project, add the code snippet below on your gradle file.

```
repositories {
   maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
}
```

Next, add the below dependency to your module's build.gradle file.

```
dependencies {
    implementation "com.github.skydoves:balloon:1.4.7-SNAPSHOT"
}
```

**Comment by ozaneb** [comment]
Unfortunately, it is the same.

**Comment by skydoves** [comment]
Hey @ozaneb, 
Could you build again with the `1.4.7-20220805.140825-9` snapshot version? Thanks!

**Comment by ozaneb** [comment]
Hey, now it works properly, thanks!

**Comment by skydoves** [comment]
@ozaneb This has been released in version `1.4.7`. Thanks for reporting this issue!
